src/py42/clients/watchlists.py

```diff
@@ -0,0 +1,204 @@
+from py42.choices import Choices
+
+
+class WatchlistType(Choices):
+    """Constants available for setting the type of watchlist.
+
+    * ``CONTRACT_EMPLOYEE``
+    * ``DEPARTING_EMPLOYEE``
+    * ``ELEVATED_ACCESS_PRIVILEGES``
+    * ``FLIGHT_RISK``
+    * ``HIGH_IMPACT_EMPLOYEE``
+    * ``NEW_EMPLOYEE``
+    * ``PERFORMANCE_CONCERNS``
+    * ``POOR_SECURITY_PRACTICES``
+    * ``SUSPICIOUS_SYSTEM_ACTIVITY``"
+    """
+
+    CONTRACT_EMPLOYEE = "CONTRACT_EMPLOYEE"
+    DEPARTING_EMPLOYEE = "DEPARTING_EMPLOYEE"
+    ELEVATED_ACCESS_PRIVILEGES = "ELEVATED_ACCESS_PRIVILEGES"
+    FLIGHT_RISK = "FLIGHT_RISK"
+    HIGH_IMPACT_EMPLOYEE = "HIGH_IMPACT_EMPLOYEE"
+    NEW_EMPLOYEE = "NEW_EMPLOYEE"
+    PERFORMANCE_CONCERNS = "PERFORMANCE_CONCERNS"
+    POOR_SECURITY_PRACTICES = "POOR_SECURITY_PRACTICES"
+    SUSPICIOUS_SYSTEM_ACTIVITY = "SUSPICIOUS_SYSTEM_ACTIVITY"
+
+
+class WatchlistsClient:
+    """A client to expose the watchlists API.
+
+    `Rest Documentation <https://developer.code42.com/api/#tag/Watchlists>`__
+    """
+
+    def __init__(self, watchlists_service):
+        self._watchlists_service = watchlists_service
+
+    def get(self, watchlist_id):
+        """Get a watchlist.
+
+        Args:
+                watchlist_id (str): A unique watchlist ID.
+
+        Returns:
+                :class:`py42.response.Py42Response`
+        """
+        return self._watchlists_service.get(watchlist_id)
+
+    def delete(self, watchlist_id):
+        """Delete a watchlist.
+
+        Args:
+                watchlist_id (str): A unique watchlist ID.
+
+        Returns:
+                :class:`py42.response.Py42Response`
+        """
+        return self._watchlists_service.delete(watchlist_id)
+
+    def get_page(self):
```

Comment:
do we think there's even a use case for paging watchlists? seems like there wouldn't ever be more than one page in most reasonable cases